.storybook/themes/commerce.scss

```diff
@@ -456,6 +456,40 @@
 				),
 			),
 			blocks: (
+				alert-bar: (
+					align-items: flex-start,
+					background-color: var(--global-red-5),
```

Comment:
This should be red-4
```suggestion
					background-color: var(--global-red-4),
```

Reply:
Using red-5 per Kelly here:
![image](https://user-images.githubusercontent.com/2287238/172949141-7e00c97e-e9fc-4ccf-91ee-622765c2fa46.png)
https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr?node-id=3507:35664#208174296

Reply:
at least, I think that's what she was saying... checking on this.

Reply:
Just confirming:
![image](https://user-images.githubusercontent.com/2287238/173094806-cd8fd217-2396-4654-8d4f-5257794256a2.png)